gateway-service/src/main/java/org/zowe/apiml/gateway/controllers/AuthController.java

```diff
@@ -122,18 +122,18 @@ public void invalidateJwtToken(HttpServletRequest request, HttpServletResponse r
     @DeleteMapping(path = ACCESS_TOKEN_REVOKE)
     @ResponseBody
     @HystrixCommand
-    public ResponseEntity<String> revokeAccessToken(@RequestBody() Map<String, String> body) throws IOException {
```

Comment:
I think we should avoid bringing changes to this version of the gateway-service, it could make it harder to merge the migration one